src/components/TokenLocking/BoostMeter.tsx

```diff
@@ -0,0 +1,37 @@
+import { FAKE_NOW } from '@/hooks/useLockHistory'
+import { Box, LinearProgress, Stack, Typography } from '@mui/material'
+import { SEASON2_START } from './BoostGraph/graphConstants'
+
+export const BoostMeter = () => {
+  const value = ((SEASON2_START - FAKE_NOW) / SEASON2_START) * 100
+
+  return (
+    <Stack direction="row" spacing={2} height="100%">
```

Comment:
```suggestion
    <Stack direction="row" spacing={2} height="100%" alignItems="flex-end">
```

To move the text to the bottom. I think a small icon above the text is also missing

Reply:
<img width="283" alt="Screenshot 2024-03-18 at 11 30 39" src="https://github.com/safe-global/safe-dao-governance-app/assets/5880855/0e2655d2-2872-4671-9b37-8e984a7375f8">

Reply:
The info icon was removed by now and I adjusted the positioning + added the icon :)